blocks/library/quote/index.js

```diff
@@ -111,11 +113,24 @@ registerBlockType( 'core/quote', {
 	},
 
 	edit( { attributes, setAttributes, focus, setFocus, mergeBlocks } ) {
-		const { value, citation, style = 1 } = attributes;
+		const { align, value, citation, style = 1 } = attributes;
 		const focusedEditable = focus ? focus.editable || 'value' : null;
 
-		return (
-			<blockquote className={ `blocks-quote blocks-quote-style-${ style }` }>
+		return [
+			focus && (
```

Comment:
I've grown more okay with it. Seems reasonable to expect that the render logic should be responsible for deciding if controls should be shown depending on whether the block currently has focus.